### DISCUSSION TITLE
How to claim ED3DAO credential?

### DISCUSSION THREAD

**Discussion by jslusser** [learningeconomy/LearnCard#189] [Help] [answered]
I recently completed the ED3DAO Intro to Decentralized Finance course and received the Nov 08 email with instructions on how to claim.  I am an iOS user, so I joined LearnCard via the web, using my @jslusser@kurkuma.co email.   I've received the LearnCard Discord message telling me "To claim the credential, you need to setup your wallet or LearnCard." and now I am stuck.  I cannot figure out what I need to do from here 🤦‍♂️.  Help?!  Discord refers me back to the docs, and the docs refer me here.  Thanks in advance.

**Comment by Custard7** [comment]
Hey @jslusser! Congrats on your Ed3 DAO Credential 🎉

To claim the credential, use the `/start-connect-id` command in the private DM you have with the LearnCard Discord bot.

![Screenshot_20221113-114853.png](https://user-images.githubusercontent.com/2185016/201533635-1b9421fa-6fa8-4b18-97da-fddff85ef0b3.png)

The bot will send you a link to connect your LearnCard to your Discord account. Click that, and follow the instructions - it will give you a verification code that you should copy to your clipboard.

Then, go back to your private DM with the LearnCard bot and complete the verification process:

Enter the slash command:

`/finish-connect-id` 

And then paste the verification code into the box! ✅

The LearnCard Discord bot will then send you any pending credentials you have earned, and you can click the Claim Link to claim your credential! 

Let me know if this answers your question 🙏

-- 
P.S. 
Thank you for asking this question - super helpful for others who may have the same question. I will update the docs to have this full explanation shortly as well 💪

**Comment by jslusser** [comment]
Have tried twice (on different days) with no bot response?  -J

￼
James Slusser
Founder

￼


Kurkuma
Florida HQ: 936 Intracoastal Drive, #17C | Ft. Lauderdale, FL 33304
California: 434 13th St Ste 404 | San Diego, CA 92101
(727) 851-6428

find us online: website <https://www.kurkuma.co/> | linkedin <https://www.linkedin.com/company/kurkuma> | twitter <https://twitter.com/kurkumaco>

**Comment by Custard7** [comment]
Hey @jslusser sorry to hear you are having issues, we are looking through the logs to find any errors or hangups.

The bot was down for maintenance earlier this week, so that may have caused additional problems.

While we look on our end, may I ask:

- to confirm, when you run the `/start-connect-id` prompt, you get a timeout error? What exactly does the bot say / respond with?
- what was the last time you tried?
- do other commands work for you, including /send-test-credential, or /finish-connect-id

Thank you for helping to debug 🙏🙏

**Comment by Custard7** [comment]
In the meantime, @jslusser if you are still having issues with the command, please send an email to sdk@learningeconomy.io, I can send you a manually-created identity link URL for you (I would send here, but it's a private link).

**Comment by Custard7** [comment, marked as answer]
For others who stumble on this thread, the issue was that the poster was copy-pasting the slash commands (`/start-connect-id`) into the Discord DM with the bot. 

When you copy-paste a command, Discord does not recognize them, and you will experience the problem noted above.

You must type the command yourself, and the command should work 💪
